free-science-books.md

```diff
@@ -1,14 +1,31 @@
 # Subject areas (In order of LIBRARY OF CONGRESS CLASSIFICATION)
 
 ## Science (General)   (Q)
+* [Fundamentals of Piano Practice](http://www.pianopractice.org/)
+
 ## Mathematics (QA)
+
 ## Astronomy   (QB)
+
 ## Physics (QC)
+* [The Feynman Lectures on Physics](http://feynmanlectures.caltech.edu/)
+* [Quantum Diaries](http://www.quantumdiaries.org/)
+
 ## Chemistry   (QD)
+
 ## Geology (QE)
+
 ## Natural history (QH)
+
 ## Botany  (QK)
+
 ## Zoology (QL)
+
 ## Human anatomy   (QM)
+
 ## Physiology (QP)
+* [Biochemical Pathways](http://biochemical-pathways.com/#/map/1)
+* [Brain Basics](http://www.nimh.nih.gov/health/educational-resources/brain-basics/brain-basics.shtml)
```

Comment:
More of web page than a book. But NLM has a great list of free books here: http://www.ncbi.nlm.nih.gov/books/browse/